os/board/rtl8720e/src/component/mbed/targets/hal/rtl8720e/us_ticker.c

```diff
@@ -63,7 +63,7 @@ uint32_t us_ticker_read(void)
 	uint64_t us_tick;
 
 	tick_cnt = SYSTIMER_TickGet(); //up counter
-	us_tick = (uint64_t)(tick_cnt * (1000000 / 32768));
```

Comment:
Due to SVACE issue (overflow), we pushed the change like this. Please confirm this does not cause it again.